---a/net/minecraft/world/gen/NoiseGenerator.java
+++b/net/minecraft/world/gen/NoiseGenerator.java
@@ -1,7 +1,7 @@
 package net.minecraft.world.gen;
 
-public abstract class NoiseGenerator
-{
+public abstract class NoiseGenerator {
+
     private static final String __OBFID = "CL_00000538";
-}
+}
